stable/datadog/templates/daemonset.yaml

```diff
@@ -45,212 +45,24 @@ spec:
-          {{- if .Values.datadog.logsEnabled }}
-          - name: DD_LOGS_ENABLED
-            value: {{.Values.datadog.logsEnabled | quote}}
-          {{- end }}
-          {{- if .Values.datadog.logsConfigContainerCollectAll }}
-          - name: DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
-            value: {{.Values.datadog.logsConfigContainerCollectAll | quote}}
-          {{- end }}
-          {{- if .Values.datadog.criSocketPath }}
-          - name: DD_CRI_SOCKET_PATH
-            value: {{ .Values.datadog.criSocketPath | quote }}
-          {{- end }}
-          {{- if not .Values.datadog.livenessProbe }}
-          - name: DD_HEALTH_PORT
-            value: "5555"
-          {{- end }}
-          {{- if .Values.datadog.useDogStatsDSocketVolume }}
-          - name: DD_DOGSTATSD_SOCKET
-            value: "/var/run/datadog/dsd.socket"
-          {{- end }}
-          {{- if and .Values.clusterAgent.clusterChecks.enabled  (not .Values.clusterchecksDeployment.enabled) }}
-          - name: DD_EXTRA_CONFIG_PROVIDERS
-            value: "clusterchecks"
-          {{- end }}
-{{- if .Values.datadog.env }}
-{{ toYaml .Values.datadog.env | indent 10 }}
-{{- end }}
-        volumeMounts:
-          {{- if .Values.datadog.useCriSocketVolume }}
-          - name: runtimesocket
-            mountPath: {{ default "/var/run/docker.sock" .Values.datadog.criSocketPath | quote }}
-            readOnly: true
-          {{- end }}
-          {{- if .Values.datadog.useDogStatsDSocketVolume }}
-          - name: dsdsocket
-            mountPath: "/var/run/datadog"
-          {{- end }}
-          - name: procdir
-            mountPath: /host/proc
-            readOnly: true
-          - name: cgroups
-            mountPath: /host/sys/fs/cgroup
-            readOnly: true
-          - name: s6-run
-            mountPath: /var/run/s6
-          {{- if (or (.Values.datadog.confd) (.Values.datadog.autoconf)) }}
-          - name: confd
-            mountPath: /conf.d
-            readOnly: true
-          {{- end }}
-          {{- if .Values.datadog.checksd }}
-          - name: checksd
-            mountPath: /checks.d
-            readOnly: true
-          {{- end }}
-          {{- if .Values.datadog.logsEnabled }}
-          - name: pointerdir
-            mountPath: /opt/datadog-agent/run
-          {{- end }}
-          {{- if .Values.datadog.logsEnabled }}
-          - name: logpodpath
-            mountPath: /var/log/pods
-            readOnly: true
-          {{- if .Values.datadog.containerLogsPath }}
-          - name: logcontainerpath
-            mountPath: {{ .Values.datadog.containerLogsPath | quote }}
-            readOnly: true
-          {{- end }}
-          {{- end }}
-          {{- if .Values.datadog.processAgentEnabled }}
-          - name: passwd
-            mountPath: /etc/passwd
-            readOnly: true
-          {{- end }}
-{{- if .Values.datadog.volumeMounts }}
-{{ toYaml .Values.datadog.volumeMounts | indent 10 }}
```

Comment:
Hello @xlucas! Just wanna confirm if this removal is intentional? If i'm not mistaken, it's still supported in our documentation? 🤔 
- https://github.com/helm/charts/blob/f8a6ecb0f17f905578e509c48c2c2f9c03ad3a59/stable/datadog/values.yaml#L217-L220

Reply:
Ah, I've overlooked [this part](https://github.com/helm/charts/pull/13088/files#diff-adf018a6543db0e2bca334a71f4ef83aR139). Seems like it's now aligned with the docs for agent container. 👍 
- https://docs.datadoghq.com/agent/kubernetes/dogstatsd/#share-the-socket-path-with-your-application

My bad!

Reply:
Yes, container-related stuff lives in different files now. So there's also [that other part](https://github.com/helm/charts/pull/13088/files#diff-bdbdb46df012256f3b6f37dd56ba6f46R189) as well.